dockerfile

```diff
@@ -3,5 +3,4 @@ WORKDIR /home/jira
 COPY requirements.txt .
 RUN pip3 install -r requirements.txt
 COPY get-events.py .
-COPY start.sh start.sh
```

Comment:
I think the lowercase version of this file is throwing something off. On my laptop i have `Dockerfile` but git is tracking `dockerfile`. so for me this line in this file doesnt exist. I would just undo this change for this commit and rename your file to have the upercase